Content.Server/Zombies/ZombieSystem.Transform.cs

```diff
@@ -101,7 +101,14 @@ public void ZombifyEntity(EntityUid target, MobStateComponent? mobState = null)
             RemComp<ThirstComponent>(target);
 
             //funny voice
-            EnsureComp<ReplacementAccentComponent>(target).Accent = "zombie";
+            var accentType = Prototype(target)!.ID switch
+            {
+                "MobMoth" => "zombieMoth",
+                "MobMothroach" => "zombieMoth",
+                _ => "zombie"
+            };
```

Comment:
Absolutely never use prototype checks, they're disallowed. Add a component to moths instead.

Reply:
Changed it to a component.